lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_generate_strings_file_from_code.rb

```diff
@@ -0,0 +1,74 @@
+require 'open3'
+
+module Fastlane
+  module Actions
+    class IosGenerateStringsFileFromCode < Action
+      def self.run(params)
+        globbed_paths = params[:paths].map { |p| File.file?(p) ? p : "#{p}/**/*.{m,swift}" }
+        files = Dir.glob(globbed_paths)
+
+        flags = [('-q' if params[:quiet]), ('-SwiftUI' if params[:swiftui])].compact
+
+        out, status = Open3.capture2e('genstrings', '-o', params[:output_dir], *flags, *files)
+
+        UI.error("genstrings failed with exit code #{status.exitstatus}") unless status.success?
+        UI.command_output(out) unless params[:quiet]
+
+        # Return the warnings as an Array
+        out.split("\n")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Generate the .strings files from your ObjC and Swift code'
+      end
+
+      def self.details
+        'Use genstrings to generate the .strings files from your ObjC and Swift code.'
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :paths,
+                                       env_name: 'FL_IOS_GENERATE_STRINGS_FILE_FROM_CODE_PATHS',
+                                       description: 'Array of directory paths to scan for `.m` and `.swift` files. The entries can contain glob patterns too.',
+                                       type: Array,
+                                       default_value: ['.']),
+          FastlaneCore::ConfigItem.new(key: :quiet,
+                                       env_name: 'FL_IOS_GENERATE_STRINGS_FILE_FROM_CODE_QUIET',
+                                       description: 'In quiet mode, genstrings will log warnings about duplicate values, but not about duplicate comments',
+                                       is_string: false, # Boolean
+                                       default_value: true),
+          FastlaneCore::ConfigItem.new(key: :swiftui,
+                                       env_name: 'FL_IOS_GENERATE_STRINGS_FILE_FROM_CODE_SWIFTUI',
+                                       description: 'Should we include SwiftUI\'s `Text()` when parsing code with `genstrings`?',
```

Comment:
Why did you chose to use single quotes as opposed to 

```suggestion
                                       description: "Should we include SwiftUI's `Text()` when parsing code with `genstrings`?",
```

Reply:
I think `rubocop` chose that for me 😄  I'll change the wording to get rid of the `?` anyway while also addressing https://github.com/wordpress-mobile/release-toolkit/pull/309/files#r752860056.

Reply:
Fixed in 2b4d317230f1383dcec12361441a8f0c52ce5265